CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing Guidelines
+
+Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
+
+Actually – before we start – a quick word of caution: Unfortunately the contribution process involves some technical knowledge. We aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
```

Comment:
Is it true though? I think if we pair both PR and having Issues templates, people should theoretically just be able to send a message through Issues (which feels a lot less daunting.)

It seems like you're setting folks up for that later in the contribution section.

Reply:
Yes. But they would still need to have a GitHub account and be comfortable to have their work discussed in the open. Especially the former creates barrier to contributions.

Reply:
That's a good point. Could we possibly add that with links to tutorials (pre-written by GitHub I imagine).

Reply:
I think so, yes. I would also create some templates, mirroring those I’ve added to the docs. 

See: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository